docs/HowToFAQ/ERROR_NoTriggers.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Warning: /!\ No terms found to trigger on."
+---
+
+This warning occurred with the following code:
+```dafny
+predicate ExistsSingleInstance(s : string32, delim : string32)
+  {
+    exists i : nat :: i <= |s| - |delim| && forall x : nat :: x < |delim| ==> s[i + x] == delim[x] &&
```

Comment:
This mostly unnamed long expressions are difficult to read. Why did you include an example that contains two types of bugs? I think having one bug per example is easier to learn from.

Reply:
The goal was to include user questions as they posed them. But I agree -- I removed the part about bad operator precedence.